testing/60-submodule.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+submodule=$(mktemp -d)
+repo=$(mktemp -d)
+cd $submodule
+git init
+git commit --allow-empty -m "Initial commit"
+cd $repo
+git init
+git submodule add $submodule submodule
+git commit --allow-empty -m "Initial commit"
+touch submodule/a
+git diff | cat
```

Comment:
@fdcds Do you know how to reproduce the "Submodule XXX contains untracked content" message? This script was my first attempt, but it yields

```diff
diff --git a/submodule b/submodule
--- a/submodule
+++ b/submodule
@@ -1 +1 @@
-Subproject commit 4def2debb586b2ae007ca8589d52efeb9a827432
+Subproject commit 4def2debb586b2ae007ca8589d52efeb9a827432-dirty
```

Reply:
This is the (at least remotely) related part of my `~/.gitconfig`:
```ini
[rebase]
#       autostash = true    
#       autosquash = true    
        stat = true
[fetch]
        prune = true
[merge]
        defaultToUpstream = yes
        log = yes
[color]
        ui = auto
[log]
        decorate = true
[diff]
        submodule = log
[status]
        submodulesummary = true
[core]
        pager = delta --dark
```

I suspect `diff.submodule = log` to be the part that causes git to output the message you are missing. Can you reproduce with that?

Reply:
Thanks, yes that's it. 9736ddb1c368e834d3f68845025c840a450b9d68